apps/ledger-live-desktop/src/renderer/screens/market/MarketCoinScreen/index.tsx

```diff
@@ -147,20 +144,19 @@ export default function MarketCoinScreen() {
 
       history.push({
         pathname: "/exchange",
-        state:
-          ptxSmartRouting?.enabled && currency?.internalCurrency
-            ? {
-                currency: currency.internalCurrency?.id,
-                mode: "buy", // buy or sell
-              }
-            : {
-                mode: "onRamp",
-                defaultTicker:
-                  currency && currency.ticker ? currency.ticker.toUpperCase() : undefined,
-              },
+        state: currency?.internalCurrency
+          ? {
+              currency: currency.internalCurrency?.id,
+              mode: "buy", // buy or sell
+            }
+          : {
+              mode: "onRamp",
+              defaultTicker:
+                currency && currency.ticker ? currency.ticker.toUpperCase() : undefined,
+            },
```

Comment:
Not sure if this else have sense anymore...is onRamp a exchange mode? or it is only buy and sell?

Reply:
> not a part of the scope of this ticket

Do we track this to update later ?

Reply:
unsure what exactly is not relative any more nor what the expected is - @sarneijim @Justkant please could you create a Jira ticket and get Xavier to prioritise?